docs/containers-registries.conf.5.md

```diff
@@ -19,6 +19,12 @@ Container engines will use the `$HOME/.config/containers/registries.conf` if it
 `credential-helpers`
 : An array of default credential helpers used as external credential stores.  Note that "containers-auth.json" is a reserved value to use auth files as specified in containers-auth.json(5).  The credential helpers are set to `["containers-auth.json"]` if none are specified.
 
+`additional-layer-store-auth-helper`
+: A string of helper name. This enables to pass registry credentials to
+  Additional Layer Store every time an image is read using the `docker://`
+  transport so that it can access private registries. See the later section for
```

Comment:
```suggestion
  transport so that it can access private registries. See the 'Enabling Additional Layer Store to access to private registries' section below for
```